index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I you're serving static files you should let Express do it for you:

1. Create a `public/` folder
2. Move every static asset there
3. Move everything in your `app/*` folder to the root (we'll reorganize later)
4. Rename your `server.js` to `app.js` to follow conventions
5. Add the following line in your `app.js` (as the first middleware!)
```js
app.use(express.static(path.join(__dirname, 'public')));  // you'll need 'path' from require('path');
```